### PR TITLE
feat: add support altitude precision on Android

### PR DESCRIPTION
**Pull request by Athorcis**
Add altitude precision on Android, to be able to use on both Android and iOS (since it is already supported on iOS)

### CODE REVIEW

File: android/common/src/main/java/com/marianhello/bgloc/data/BackgroundLocation.java [hunk 1]
```diff
@@ -811,6 +846,14 @@ public String toString () {
         } else {
             s.append(" acc=???");
         }
+
+        if (hasVerticalAccuracy) {
+            s.append(String.format(" acc=%.0f", verticalAccuracy));
```
Comment (HarelM) [inline android/common/src/main/java/com/marianhello/bgloc/data/BackgroundLocation.java]: String should be different than regular accuracy I suppose.

**Comment by HarelM** [comment]
I didn't see any defaults, what would happen when upgrading an existing database?

**Comment by Athorcis** [comment]
> I didn't see any defaults, what would happen when upgrading an existing database?

I assumed that the new columns would be zerofilled but I recon it would be better to have explicit defaults

**Review by HarelM — APPROVED** [review]
THANKS!!

**Comment by HarelM** [comment]
Feel free to open a PR with version change in order to shorten the release cycle.

**Comment by Athorcis** [comment]
> Feel free to open a PR with version change in order to shorten the release cycle.

Should I update the minor version number or just the patch version?

**Comment by HarelM** [comment]
Minor I think.

**Comment by HarelM** [comment]
@Athorcis have you ran this code?
There was a PR here to fix basic syntax that means that this code wasn't ran before (the database update code).
Did you add it after testing?
I'm not sure what I should be doing with this code if you haven't tested it... The only solution I can think of is reverting this code which I would like to avoid.

**Comment by Athorcis** [comment]
@HarelM I ran the code of the first commit (which is currently in production), but not the code of the subsequent commits and accordingly [6edc070](https://github.com/HaylLtd/cordova-background-geolocation-plugin/pull/176/commits/6edc0706b751ce43ef6fc94c5e35374b0fcf0318) seems to be the one introduce a bug

**Comment by HarelM** [comment]
Ok, thanks for the info!
If you happen to run the latest code from main and it works as expected let me know and I'll release a new version.

**Pull request merged**